package curs_java.ecercicis.exception2;

import curs_java.Universitat.practica2.EstatuetSalmeronIsaac_POO_Praactica2.src.Main;
import curs_java.ecercicis.herencia.exercici1.main;

public class prova {
    public static void main(String[] args) {

        try {

        } catch (Pizza_error e) {
            // TODO: handle exception
        }

    }
}
